image/rhel/Dockerfile

```
@@ -45,7 +45,7 @@ RUN ln -s entrypoint-wrapper.sh /stackrox/admission-control && \
     rm /tmp/snappy.rpm RPM-GPG-KEY-CentOS-Official && \
     # (Optional) Remove line below to keep package management utilities
     rpm -e --nodeps $(rpm -qa curl '*rpm*' '*dnf*' '*libsolv*' '*hawkey*' 'yum*') && \
-    rm -rf /var/cache/dnf && \
+    rm -rf /var/cache/yum && \
```

I did this in order to stay independent from the underlying ubi type

```suggestion
    rm -rf /var/cache/dnf /var/cache/yum && \
```